### PR TITLE
01 set sauce credentials fix 04 05 19

### PR DESCRIPTION
**Pull request by Tacktician**
-updated `FullJourneyTest`
-updated `exercise-guides`

### CODE REVIEW

File: src/test/java/exercises/FullJourneyTest.java [hunk 1]
```diff
@@ -19,24 +19,25 @@
...
 
-        //sets browser to Safari
-        capabilities.setCapability("browserName", "Safari");
+        //sets browser to Firefox
+        capabilities.setCapability("browserName", "firefox");
 
         //sets operating system to macOS version 10.13
         capabilities.setCapability("platform", "macOS 10.13");
 
         //sets the browser version to 11.1
```
Comment (nadvolod) [inline src/test/java/exercises/FullJourneyTest.java:33]: @spider-sauce you can just set this to "latest" in the comment and the code. That way, you don't have to update the code or the comment if we change browsers

File: exercise-guides/exercise1.md [hunk 2]
```diff
@@ -101,14 +101,22 @@
     driver.findElement(By.cssSelector(loginBtn)).click();
```
Comment (nadvolod) [inline exercise-guides/exercise1.md]: Although it's not in this PR, this part is no longer relevant https://screencast.com/t/KBr1q4dAk . We don't need the method or the exception at this point. If you want to KISS, just tell them to create a test method called `ShouldBeAbleToLogin` and you don't need to supply the code

File: src/test/java/exercises/FullJourneyTest.java (hunk 1 above)
Comment (Tacktician) [reply src/test/java/exercises/FullJourneyTest.java:33]: Ok. Will that affect anything if we don't specify `w3c` as a capability? Or do we only have to explicitly call it out in `chrome`?

File: exercise-guides/exercise1.md (hunk 2 above)
Comment (Tacktician) [reply exercise-guides/exercise1.md]: 👍

File: src/test/java/exercises/FullJourneyTest.java (hunk 1 above)
Comment (nadvolod) [reply src/test/java/exercises/FullJourneyTest.java:33]: @spider-sauce I meant for the browser version. I think I commented on the wrong line :(
"browserVersion", "latest"

**Pull request merged**